core/build.gradle.kts

```diff
@@ -118,6 +118,9 @@ kotlin {
         kotlin.srcDir("$targetName/${suffix ?: "src"}")
         resources.srcDir("$targetName/${suffix?.let { it + "Resources" } ?: "resources"}")
         languageSettings {
+            compilerOptions{
```

Comment:
This DSL block doesn't belong to this scope. I fixed it in a different way in #319, however I can try to use global compilerOptions for that.